galerautils/src/gu_threads.h

```diff
@@ -18,6 +18,7 @@ extern gu_pfs_instr_cb_t pfs_instr_callback;
 #if __unix__
 
 #include <pthread.h>
+#include <string.h>
```

Comment:
Is this change required?

Reply:
I've reverted this change. Now it is the same as upstream.
This include is needed if `GU_DEBUG_MUTEX` is defined. Also, there is a problem in `Mutex::void disown()` method, as it references the unknown variable `value` instead of `value_`.
As I can not recall the case when we compiled Galera with GU_DEBUG_MUTEX I've reverted this change to not diverge from the upstream.